### DISCUSSION TITLE
Grand Theft Auto : San Andreas - The Definitive  Edition

### DISCUSSION THREAD

**Discussion by MastermindReaper** [multitheftauto/mtasa-blue#2420] [General] [answered]
Will you guys be adding the GTA SA The Definitive Edition to MTA?  LOOKS SICK!!!! 

https://www.rockstargames.com/newswire/article/4kaa7414o5k979/grand-theft-auto-the-trilogy-the-definitive-edition-coming-november-11

**Comment by Disinterpreter** [comment]
> 
> 
> Will you guys be adding the GTA SA The Definitive Edition to MTA? LOOKS SICK!!!!
> 
> https://www.rockstargames.com/newswire/article/4kaa7414o5k979/grand-theft-auto-the-trilogy-the-definitive-edition-coming-november-11

We will consider it only after getting binary files.

**Reply by Vadya963** [reply]
They killed our boys, please don't

**Comment by websharik** [comment]
Whats news ? GTA SA Definitive is released.

**Reply by Disinterpreter** [reply]
> 
> 
> Whats news ? GTA SA Definitive is released.

Unreal Engine confirmed. If somebody rewrite `game_sa` and `multiplayer_sa`, it can be :)

**It wouldn't be soon.**

**Comment by DartPower** [comment]
I too very want a FLOSS variant of Trilogy Multiplayer mod

Not too much early was released a some mod, but it's proprietary and author don't like to publish source code at FLOSS Licensing.

```
Q: Open source?
A: No. I use a lot of my personal earnings and I don't want to give them to someone for nothing. I will try to fix bugs and flaws as soon as I can, don't worry!
```

I think community needs more open source and libre mods. More in MP case.

**Reply by PlatinMTA** [reply]
I don't think you will see an open source GTA Trilogy MP any time soon. MTA only went opensource a few years later.

**Reply by AlexTMjugador** [reply]
[TMP (Trilogy Multiplayer)](https://trilogymp.com/) is the spiritual successor to MTA for the trilogy. Some MTA contributors are working in it (although they are different projects, unaffiliated with each other), and it will eventually be open source.

**Comment by patrikjuvonen** [comment, marked as answer]
We decided to discontinue the multiplayer project for the Definitive Edition.
